garage/theano/algos/ddpg.py

```diff
@@ -272,12 +273,12 @@ def init_opt(self):
         policy_updates = self.policy_update_method(
             policy_reg_surr, self.policy.get_params(trainable=True))
 
-        f_train_qf = ext.compile_function(
+        f_train_qf = tensor_utils.compile_function(
```

Comment:
Since we want to keep all the Theano-dependent code in garage.theano, and the module garage.misc.ext had both independent and Theano-independent code, I moved all the latter code into the new module garage.theano.misc.tensor_utils to create the separation.
This is a particular case where not the whole module but only parts of it are only used by the Theano branch, while the other parts are used by both TensorFlow and Theano.

Reply:
In generally we have like O(2^n) util.py files when we should really have like O(n), so i'm in favor of merging whichever ones make any sense.